.circleci/config.yml

```diff
@@ -11,6 +11,7 @@ jobs:
           steps:
             - run: npm install
             - run: npm test
+            - run: npm run lint
```

Comment:
A tiny tip from my own frustration on other projects! 

As the project grows, the tests begin to take up more and more and more time.
Eventually they can sometimes become quite slow.

linting generally remains quick for ever! (as long as we remember to exclude the node_modules)

so its usually better to run the linting check before the tests
that way we fail quickly and can fix and retry! :)

Reply:
node_modules has been excluded in tsconfig.json
I have changed the order the tests are run in config.yml . We are now running  linting check before the tests